### ISSUE TITLE
Auto fight doesn't work?

### ISSUE THREAD

**Issue by v3n0m3k** [tantinevincent/Onegai-oyodosan#26]
Hi,
I have a problem with auto fighter. When i activate it from 'main' all it does perform is upto 'fight_checker' after that it just goes back to home port even with undamaged fleet and normal morale.

**Comment by tantinevincent** [comment]
Do you have any 間宮(mamiya)?
Mamiya is used to check fleet's morale, so it contains a bug if you don't have any one,
I opened the issue but not fix yet.

**Comment by v3n0m3k** [comment]
I do have both Mamiya and Irako.

**Comment by tantinevincent** [comment]
Could you paste the log and screenshot (remeber remove your name) for me?

**Comment by v3n0m3k** [comment]
Is there some kind of limit you can set loop? Because it started working when i changed it from 1000 to 100.

**Comment by tantinevincent** [comment]
Could you paste the modified file on gist?

https://gist.github.com/

**Comment by v3n0m3k** [comment]
I imagine you mean 'main.py' so here: https://drive.google.com/open?id=0B9dE2OuHvvSEcHZVYUhGa2lkTDQ&authuser=0

**Comment by tantinevincent** [comment]
I can't open this file, please give permission to me.

**Comment by v3n0m3k** [comment]
Yeah, sorry. Pasted wrong link. https://drive.google.com/file/d/0B9dE2OuHvvSEcHZVYUhGa2lkTDQ/view?usp=sharing

**Comment by tantinevincent** [comment]
In main.py,  "count" is just a variable about loop times. I think this modified variable should not change anything. 

According to the following issue, I guess this problem comes from sikuli: 

https://bugs.launchpad.net/sikuli/+bug/769924
https://answers.launchpad.net/sikuli/+question/262328
